packages/utilities/psammead-styles/src/__snapshots__/fonts.test.js.snap

```diff
@@ -46,6 +81,17 @@ exports[`Psammead Styles - Fonts should match LATHA REGULAR base font url 1`] =
   }"
 `;
 
+exports[`Psammead Styles - Fonts should match LATHA REGULAR with overridden url 1`] = `
```

Comment:
I may be missing something in the diff, but would you mind clarifying what's caused these block additions for fonts unrelated to Qalam?

Reply:
I updated the test to have a better desctription in the snapshot https://github.com/bbc/psammead/blob/af4c0b40910e7a7e765c9ebd24d782dc3f9827ab/packages/utilities/psammead-styles/src/fonts.test.js#L16

Reply:
Previously it was https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-styles/src/fonts.test.js#L16

Reply:
Aha! Thanks.